.rubocop.yml

```diff
@@ -38,6 +38,10 @@ Style/NumericLiteralPrefix:
 Style/TernaryParentheses:
   Enabled: false
 
+# sometimes it is usefull to have those empty methods
```

Comment:
yeah `-a` would fix it anyway 👍  i am not a fan of it, and i thought that i saw some other rules  like that.